contracts-periphery/script/Deploy.s.sol

```diff
@@ -2,9 +2,15 @@
 pragma solidity ^0.8.16;
 
 import "forge-std/Script.sol";
+import {UmbraBatchSend} from "src/UmbraBatchSend.sol";
+import {IUmbra} from "src/interface/IUmbra.sol";
 
 contract Deploy is Script {
+  address constant UMBRA = 0xFb2dc580Eed955B528407b4d36FfaFe3da685401;
+
   function run() public {
     vm.broadcast();
+    UmbraBatchSend batchSend = new UmbraBatchSend(IUmbra(UMBRA));
+    console2.log("batchSend Address: ", address(batchSend));
```

Comment:
Probably can remove this line, deploys are logged by default. You can see the address once in the logs section and once in the spot where the tx was sent. Otherwise PR LGTM!

![image](https://user-images.githubusercontent.com/17163988/202038301-a121961f-331f-4480-aee8-a81b94b7f3f5.png)

Reply:
Thanks for the review! Removed console log. And was able to deploy to Goerli for test purposes 0xDf4B5B2C08A77078c05176D8dFAb6274D54215e9